### PR TITLE
Test - Update for MC 1.19+

### PR DESCRIPTION
**Pull request by ShaneBeee**
### Description
This PR aims to update tests by removing outdated version checks since Skript 2.10+ will be only for MC 1.19.4+

### NOTES:
- ~~`boats.sk` was failing, so I temporarily commented out the line until that can be fixed (so tests can pass)~~
Twas an Alias issue, Pickle informed me to update Aliases submodule on local and now it works.
- `4769-fire-visualeffect.sk` was failing. I think the fire block was being set mid air and wouldn't set, so I just added a block below it
- `pull-5566...sk` was failing, it was getting the wrong blocks (probably edited in another test), so I shifted the test over 10 blocks along Z

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/test/skript/tests/regressions/pull-5566-block iterator being 100 always.sk [hunk 1]
```diff
@@ -3,5 +3,7 @@
 test "100 blocks fix":
 	set {_l} to test-location ~ vector(0, -1, 0)
 	set block at {_l} to air
-	assert blocks 5 below {_l} contains air, grass block, dirt block, dirt block, bedrock block and void air with "Failed to get correct blocks (got '%blocks 5 below test-location%')"
+
+	# Failing due to finding different set of blocks
+	# assert blocks 5 below {_l} contains air, grass block, dirt block, dirt block, bedrock block and void air with "Failed to get correct blocks (got '%blocks 5 below test-location%')"
```
Comment (Efnilite) [inline src/test/skript/tests/regressions/pull-5566-block iterator being 100 always.sk:8]: can probably be fixed by setting the blocks manually before checking

**Review by Efnilite — APPROVED** [review]

File: src/test/skript/tests/regressions/pull-5566-block iterator being 100 always.sk (hunk 1 above)
Comment (ShaneBeee) [reply src/test/skript/tests/regressions/pull-5566-block iterator being 100 always.sk]: I ended up just offsetting with the vector, 10 on the z axis.
I think the issue is probably another test script is manipulating the blocks at that position.

**Comment by TheAbsolutionism** [comment]
I could be wrong, but I believe the test added from Equip's EffDetonate is the cause of those tests failing. As the explosion that's being detonated is at spawn where those 2 tests take place.

**Review by sovdeeth — APPROVED** [review]
nice work

File: src/test/skript/tests/syntaxes/expressions/ExprLowestHighestSolidBlock.sk [hunk 2]
```diff
@@ -1,34 +1,4 @@
...
-	set block at location(0.5, 2.5, 0.5, "world") to dirt
-	set {_lowest} to lowest solid block at location(0.5, 64, 0.5, "world")
-	assert type of {_lowest} is dirt with "lowest block is not dirt (got '%type of {_lowest}%')"
-	assert location of {_lowest} is location(0.5, 2.5, 0.5, "world") with "lowest block is not at 0.5,2.5,0.5 (got '%location of {_lowest}%')"
-	set block at location(0.5, 0.5, 0.5, "world") to {_oldBlock::1}
-	set block at location(0.5, 1.5, 0.5, "world") to {_oldBlock::2}
-	set block at location(0.5, 2.5, 0.5, "world") to {_oldBlock::3}
-
-test "lowest/highest solid block (new height)" when running minecraft "1.18":
+test "lowest/highest solid block (new height)":
```
Comment (APickledWalrus) [inline src/test/skript/tests/syntaxes/expressions/ExprLowestHighestSolidBlock.sk:1]:
```suggestion
test "lowest/highest solid block":
```

**Review by APickledWalrus — APPROVED** [review]

**Pull request merged**